vendingMachine.js

```diff
@@ -1,68 +1,90 @@
 const prompt = require('prompt-sync')();
-let products = [
+const productsMachine = [
   ["Chocorramo", "Gansito", "Galletas"],
   ["Maizitos", "Tostacos", "Yogurt"],
   ["Papas", "Platanitos", "Ponque"]
 ];
-let stock = [
+const productStock = [
   [5, 5, 5],
   [5, 5, 5],
   [5, 5, 5]
 ];
 
 function showVendingMachine(arr1, arr2) {
-  let result = '';
-  let coord, stoc;
+  const space = " ";

```

Comment:
Uppercase for constants in format SPACE, SPACE_TABLE

Reply:
done